scripts/build_llvm.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+if [[ -n "$CONDA_DEFAULT_ENV" ]]; then
+    echo "Conda environment is active. Please deactivate Conda and/or Sparta before continuing."
+    exit 1
+fi
+
+if [ -z "$TOP" ] || [ ! -d "$TOP" ]; then
+    echo "The TOP variable is not set to a valid directory."
+    echo "To set the required environment variables, cd into your work area and run: source how-to/env/setuprc.sh"
+    exit 1
+fi
+
+read -p "Enter the install path [default is $TOP]: " INSTALL_PATH
+INSTALL_PATH=${INSTALL_PATH:-$TOP}
+
```

Comment:
jeffnye@compute2:/data/users/jeffnye/condor$ bash how-to/scripts/build_llvm.sh
Enter the install path [default is /data/users/jeffnye/condor]: 

That is not where it gets installed, it is actually  /data/users/jeffnye/condor/llvm

This is not hard, get this right and so this task can be closed.